controllers/certificate_lifecycle_controller.go

```diff
@@ -35,28 +35,31 @@ type CertificateLifecycle struct {
 }
 
 func (s *CertificateLifecycle) Reconcile(ctx context.Context, request reconcile.Request) (reconcile.Result, error) {
-	logger := log.FromContext(ctx)
+	log := log.FromContext(ctx)
```

Comment:
It's not a best practice having a variable name as the package one since it could lead to confusion. If we're using this approach we could correct the "wrong" one.

Reply:
So let's rename everything to logger ?

Reply:
May I ask you to address this in a separate PR, please?

Reply:
I agree it's better, I will do it.